Fix Typo in phpMyAdmin example

randallmorse opened the pull request:
the service is mysql not some-mysql

<!-----------------------------------------------------------------------------
Thank you for contributing to the PrestaShop developer documentation! 

Please take the time to edit the "Answers" rows below with the necessary information.

Check out our contribution guidelines on how to contribute:
https://devdocs.prestashop-project.org/8/contribute/documentation/how/
------------------------------------------------------------------------------>

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       |  8.x 
| Description?  | The docker compose example for adding phpMyAdmin has a typo in its links definition, the example fails as its looking for the service name instead of the container name, additionally ensure the the container can reach the mysql service by adding it to the shared private network.

<!-- Click the form's "Preview" button to make sure the table is functional in GitHub. Thank you! -->

randallmorse committed:
Fix Typo in PHPMyAdmin example

the service is mysql not some-mysql

SharakPL commented on basics/installation/environments/docker.md:
```diff
@@ -339,7 +339,7 @@ services:
-        - some-mysql
+        - mysql
       environment:
         PMA_HOST: some-mysql
```
```suggestion
        PMA_HOST: mysql
```
This also should be changed to `mysql` as the host ~~must match~~ the database service name too.

Edit: actually it looks like it works both ways. Either service name or container name will link phpmyadmin properly.

randallmorse committed:
The network should be defined or the container wont be able to reach the mysql service 

The network should be defined or the container wont be able to reach the mysql service unless you expose the ports. This is likely the best example.

randallmorse committed:
Additional example configuration for phpMyAdmin

with the PMA_HOST and PMA_PORT and PMA_ARGITRARY defined it seems that the intent is to have the system leverage this connection information, however with out the PMA_USER and the PMA_PASSWORD configuration set this information is not used, to better serve the example the db user and pass from  lines 25 and 26 could be set here and then the container would auto login for phpMyAdmin for the example

randallmorse commented on basics/installation/environments/docker.md:
```diff
@@ -339,14 +339,18 @@ services:
         PMA_PORT: 3306
         PMA_ARBITRARY: 1
+        PMA_USER: root
+        PMA_PASSWORD: admin
```
@SharakPL is this the intent of having the host name and port set here? see comment on the commit, at current it isnt used unless the the PMA_HOST and PMA_PASSWORD is set, I think this might have been the intent and if not could be a great add to the example to assist new users.

kpodemski commented:
thank you @randallmorse
